tests/generative/variable_strategies.py

```diff
@@ -96,31 +104,78 @@ def select_column(type_, frame):
         column_names = [n for n, t in schema.column_types if t == type_]
         return st.builds(SelectColumn, st.just(frame), st.sampled_from(column_names))
 
+    @st.composite
+    def is_null(draw, _type, frame):
+        type_ = draw(any_type())
+        return Function.IsNull(
+            draw(series(type_, draw(one_row_per_patient_frame_or(frame))))
```

Comment:
This one already existed and is (mostly) just reordered so the functions are in the same order they appear in `series_constraints`.  It also now passes `draw(one_row_per_patient_frame_or(frame))` to `series` instead of just `frame` - because I'd misunderstood what `one_row_per_patient_frame_or` was doing before and thought it was always generating one-row-per-patient